spec/bundle/cask_dumper_spec.rb

```diff
@@ -36,13 +36,13 @@
     end
   end
 
-  context "cask `foo`, `bar` and `baz` are installed, while `baz` is required by formula" do
+  context "when casks `foo`, `bar` and `baz` are installed, with `baz` being a formula requirement" do
     before do
       described_class.reset!
 
-      foo = instance_double("Cask::Cask", to_s: "foo")
-      bar = instance_double("Cask::Cask", to_s: "bar")
-      baz = instance_double("Cask::Cask", to_s: "baz")
+      foo = instance_double("Cask::Cask", to_s: "foo", config: nil)
+      bar = instance_double("Cask::Cask", to_s: "bar", config: double(explicit: { fontdir: "/Library/Fonts" }))
```

Comment:
`brew style` says "Prefer using verifying doubles over normal doubles", so how would I change this so a call to `cask.config.explicit` still returns something?

Reply:
Does https://relishapp.com/rspec/rspec-mocks/docs/verifying-doubles help?